toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -369,14 +459,14 @@ func convertInputImageToWriteableFormat(ctx context.Context, ic *ImageCustomizer
 
 		inputIsoArtifacts, err := createIsoArtifactStoreFromIsoImage(ic.inputImageFile, filepath.Join(ic.buildDirAbs, "from-iso"))
 		if err != nil {
-			return inputIsoArtifacts, fmt.Errorf("failed to create artifacts store from (%s):\n%w", ic.inputImageFile, err)
+			return inputIsoArtifacts, fmt.Errorf("%w (source='%s'):\n%w", ErrCreateArtifactsStore, ic.inputImageFile, err)
```

Comment:
`ErrCreateArtifactsStore` is related to collection artifacts for generating an ISO. So, it should probably be in the `LiveOS` category.

Reply:
updated to LiveOS category